opentelemetry/src/test/java/io/grpc/opentelemetry/OpenTelemetryMetricsModuleTest.java

```diff
@@ -0,0 +1,1065 @@
+                            histogram.hasPointsSatisfying(
+                                point ->
+                                    point
+                                        .hasCount(1)
+                                        .hasSum(0.100)
+                                        .hasAttributes(clientAttributes1),
+                                point ->
+                                    point
+                                        .hasCount(2)
+                                        .hasSum(0.154 + 0.032)
+                                        .hasAttributes(clientAttributes),
+                                point ->
+                                    point
+                                        .hasCount(1)
+                                        .hasSum(0.024)
+                                        .hasAttributes(clientAttributes2))),
+            metric ->
+                assertThat(metric)
+                    .hasInstrumentationScope(InstrumentationScopeInfo.create(
+                        OpenTelemetryConstants.INSTRUMENTATION_SCOPE))
+                    .hasName(
+                        OpenTelemetryConstants.CLIENT_ATTEMPT_RECV_TOTAL_COMPRESSED_MESSAGE_SIZE)
+                    .hasUnit("By")
+                    .hasHistogramSatisfying(
+                        histogram ->
+                            histogram
+                                .isCumulative()
+                                .hasPointsSatisfying(
+                                    point ->
+                                        point
+                                            .hasCount(1)
+                                            .hasSum(0)
+                                            .hasAttributes(clientAttributes1),
+                                    point ->
+                                        point
+                                            .hasCount(2)
+                                            .hasSum(0 + 0)
+                                            .hasAttributes(clientAttributes),
+                                    point ->
+                                        point
+                                            .hasCount(1)
+                                            .hasSum(33D)
+                                            .hasAttributes(clientAttributes2))));
+  }
+
+
+
+  @Test
+  public void clientStreamNeverCreatedStillRecordMetrics() {
+    OpenTelemetryState state = registerTestInstruments(testMeter, true, false);
+    OpenTelemetryMetricsModule module =
+        new OpenTelemetryMetricsModule(fakeClock.getStopwatchSupplier(), state);
+    OpenTelemetryMetricsModule.CallAttemptsTracerFactory callAttemptsTracerFactory =
+        new OpenTelemetryMetricsModule.CallAttemptsTracerFactory(module,
+            method.getFullMethodName());
+    ClientStreamTracer streamTracer =
+        callAttemptsTracerFactory.newClientStreamTracer(STREAM_INFO, new Metadata());
```

Comment:
nit: this still creates new stream, defeating the test method name.

Reply:
Updated the test.